testlapack/dgebal.go

```diff
@@ -24,15 +23,9 @@ func DgebalTest(t *testing.T, impl Dgebaler) {
 
 	for _, job := range []lapack.Job{lapack.None, lapack.Permute, lapack.Scale, lapack.PermuteScale} {
 		for _, n := range []int{0, 1, 2, 3, 4, 5, 6, 10, 18, 31, 53, 100} {
-			var pNonzero float64
```

Comment:
Oops, this somehow leaked into this PR. Sorry. Anyway, specifying the number of non-zeros is much better than the probability.